lib/ansible/modules/network/cnos/cnos_rollback.py

```diff
@@ -183,6 +190,70 @@
 from collections import defaultdict
 
 
+# Utility Method to rollback the running config or start up copnfig
+# This method supports only SCP or SFTP or FTP or TFTP
+def doConfigRollBack(module, prompt, answer):
```

Comment:
It appears this code was originally (in it's refactored form) licensed as BSD.  I believe you will need to either include license information for this function stating that this function is still BSD licensed, or get approval from the prior authors of the removed code indicated above, to relicense as GPLv3.

Reply:
I can see attribution for 2 users, across the code you are removing:

dkasberg - https://github.com/ansible/ansible/commit/dbb452100ab686b4dd0bc8835425a512a785212f
amuraleedhar - https://github.com/ansible/ansible/commit/ffd701c8d1b77ff3384cf7d0ba6aa58c8fea0636

I'm an unsure as to whether you are permitted to make a relicensing decision on behalf of dkasberg as a Lenovo employee.

Reply:
David Kasberg has retired from Lenovo and is no more in world of coding. We are still very much in contact. Should I contact him for approval?.

Reply:
cc @gundalow

Reply:
@amuraleedhar That sounds good. Can you please get this PR updated do we can merge before community freeze on 30th?